.github/workflows/release.yaml

```diff
@@ -12,11 +12,14 @@ jobs:
     runs-on: ubuntu-18.04 # 20.04 doesn't provide createrepo for now
     env:
       DOCKER_CLI_EXPERIMENTAL: "enabled"
+    permissions:
+      id-token: write
     steps:
       - name: Install dependencies
         run: |
           sudo apt-get -y update
           sudo apt-get -y install rpm reprepro createrepo distro-info
+      - uses: sigstore/cosign-installer@v2.0.0
```

Comment:
I would recommend pin-by-SHA https://github.com/ossf/scorecard/blob/main/docs/checks.md#pinned-dependencies

Reply:
fixed, thank you